functions_and_scope_lab.js

```diff
@@ -0,0 +1,363 @@
+// # Function Exercises
```

Comment:
Good assignment over all. No major feedback other one minor feedback. Keep up the good work!